test/logging_test.rb

```diff
@@ -0,0 +1,77 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class LoggingTest < Minitest::Test
+      class TestLogger < ActiveSupport::Logger
+        def initialize
+          @file = StringIO.new
+          super(@file)
+        end
```

Comment:
re naming, to me

`ActiveJob::Base.logger` corresponds to `ActiveModelSerializers.logger`

and

`ActiveJob::Logging` to `ActiveModelSerializers::Logging`